src/chat-statistics/stats.py

```diff
@@ -0,0 +1,96 @@
+import json
+from pathlib import Path
+from typing import Union
+
+import arabic_reshaper
+from loguru import logger
+from bidi.algorithm import get_display
+from hazm import Normalizer, word_tokenize
+from src.data import DATA_DIR
+from wordcloud import WordCloud
+
+
+class ChatStatistics:
+    """Generates chat word cloud from a telegram chat json file.
+    """
+    def __init__(self, chat_json: Union[str, Path]) -> None:
+        """
+        :param chat_json: telegram chat json file
+        """
+        logger.info(f"Loading chat dat from {chat_json}")
+        # load chat data
+        with open(chat_json, 'r') as f:
+            self.chat_data = json.load(f)
+
+        self.normalizer = Normalizer()
+
+        # load stopwords
+        logger.info(f"Loading stopwords from {DATA_DIR / 'stopwords.txt'}")
+        stop_words = open(DATA_DIR / 'stopwords.txt').readlines()
+        stop_words = list(map(str.strip, stop_words))
+        self.stop_words = list(map(self.normalizer.normalize, stop_words))
+
+    def process_text(self, text: str) -> str:
+        """Filters and normalizes input text
+
+        :param text: input text to be processed
+        :return: processed text
+        """
+        tokens = word_tokenize(text)
+        content = ' '.join(
+            list(filter(lambda item: item not in self.stop_words, tokens)))
+        content = self.normalizer.normalize(content)
+        return content
+
+    def generate_word_cloud(self, output_dir: Union[str, Path]):
```

Comment:
Better to add word cloud arguments such as `width`, `height` and `background_color`.
```suggestion
    def generate_word_cloud(self, 
    width=800,
    height=600,
    background_color='white',
    output_dir: Union[str, Path]):